src/main/java/fr/jcgay/maven/plugin/buildplan/AbstractLifecycleMojo.java

```diff
@@ -34,6 +40,10 @@
     /** Allow to specify which tasks will be used to calculate execution plan. */
     @Parameter(property = "buildplan.tasks", defaultValue = "deploy")
     private String[] tasks;
+    
+    /** Allow to specify an output file to bypass console output */
+    @Parameter(property = "buildplan.outputFile")
+    private String outputFile;
```

Comment:
I think we should use a `File` field there (https://maven.apache.org/guides/plugin/guide-java-plugin-development.html#Parameters).  
It will allow tu use this functionality with multi-modules project.  
At the moment the `outputFile` contains only the last module execution result.

WDYT ?